libraries/standard/mqtt/cbmc/proofs/MQTT_GetSubAckStatusCodes/Makefile

```diff
@@ -0,0 +1,17 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Can you please copy the license from another proof's makefile

Reply:
Added